packages/spark/base/inputs/_select.scss

```diff
@@ -15,6 +15,12 @@
   &:focus {
     border-color: $sprk-select-focus-border-color;
   }
+
+  // Remove FF specific focusring
+  &:-moz-focusring {
```

Comment:
Fixes an inconsistency with styles. Removes the grey focusring border that FF applies that is not needed as we supply a focus style and this only shows up in FF and not other browsers. The focusring in FF can be seen below. Now this is removed.
<img width="530" alt="Screen Shot 2019-09-11 at 4 45 25 PM" src="https://user-images.githubusercontent.com/2117593/64733947-b6a4d100-d4b3-11e9-865a-428148650bd9.png">